lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = ActiveModel::Serializer::IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
No, because we want the flexibility to have different default values per adapter. For instance, the JsonApi adapter defaults to none (since the *relationship objects* themselves will be serialized anyways, we don't want to enforce the *related resources* to be included in the answer unless specified by the user).